DataScience/GenevaLogger.py

```diff
@@ -0,0 +1,48 @@
+from fluent import asyncsender as sender
+import json
+import sys
+import traceback
+
+class Logger:
+    app_id = ""
+    job_id = ""
+
+    @staticmethod
+    def create_logger(app_id, job_id):
+        sender.setup('microsoft.cloudai.personalization', host='localhost', port=24224)
```

Comment:
Make this an argument so the repository isn't dependent on personalization. We shouldn't have personalizer specific code in the repo.